x/genutil/module.go

```diff
@@ -63,20 +63,26 @@ func (AppModuleBasic) GetQueryCmd(clientCtx client.Context) *cobra.Command { ret
 type AppModule struct {
 	AppModuleBasic
 
-	accountKeeper types.AccountKeeper
-	stakingKeeper types.StakingKeeper
-	deliverTx     deliverTxfn
+	accountKeeper   types.AccountKeeper
+	stakingKeeper   types.StakingKeeper
+	deliverTx       deliverTxfn
+	txJSONDecoder   sdk.TxDecoder
+	txBinaryEncoder sdk.TxEncoder
```

Comment:
Would it be possible to just use `txGenerator client.TxGenerator` here? And pass this object around in functions?